piControlMain.c

```diff
@@ -1447,10 +1424,7 @@ static long piControlIoctl(struct file *file, unsigned int prg_nr, unsigned long
 
 	case KB_CONFIG_STOP:	// for download of configuration to Master Gateway: stop IO communication completely
 		{
-			if (piDev_g.machine_type != REVPI_CORE &&
-			    piDev_g.machine_type != REVPI_CORE_SE &&
-			    piDev_g.machine_type != REVPI_CONNECT &&
-			    piDev_g.machine_type != REVPI_CONNECT_SE) {
+			if (!piDev_g.revpi_gate_supported) {
```

Comment:
Good job restricting this to revpi_gate_supported instead of pibridge_supported.